lms/services/email_preferences.py

```diff
@@ -60,6 +74,24 @@ def h_userid(self, url: str) -> str:
         except (ExpiredJWTError, InvalidJWTError) as err:
             raise InvalidTokenError() from err
 
+    def get_preferences(self, h_userid):
+        preferences = self._user_preferences_service.get(h_userid)
+
+        for key in self.DAY_KEYS:
+            preferences.preferences.setdefault(key, True)
+
+        return {key: preferences.preferences[key] for key in self.DAY_KEYS}
```

Comment:
Return only the user's email preferences, not any other preferences they might have. This is handy for putting only the email preferences in template variables or in a JSON response etc.

We're returning a different dict here, so any modifications to the returned preferences will not be saved do the DB.